sql/memo/rel_props.go

```diff
@@ -263,64 +260,64 @@ func (p *relProps) populateFds() {
 	p.fds = fds
 }
 
+func CardMemoGroups(g *ExprGroup) {
+	if g.RelProps.stat != nil {
+		return
+	}
+	// lookup joins and index scans will be first, if any
```

Comment:
Can you add context for this comment? Why does the order of the children matter here?